cluster/gce/config-test.sh

```diff
@@ -272,6 +278,7 @@ fi
 CLUSTER_DNS_CORE_DNS="${CLUSTER_DNS_CORE_DNS:-true}"
 ENABLE_CLUSTER_DNS="${KUBE_ENABLE_CLUSTER_DNS:-true}"
 DNS_SERVER_IP="10.0.0.10"
+LOCAL_DNS_IP="169.254.25.10"
```

Comment:
Is there a reason not to allow this to be overridden in the test environment?, e.g. `LOCAL_DNS_IP="${KUBE_LOCAL_DNS_IP:-169.254.25.10}"`

Reply:
Good idea, will do.